### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CheezItMan**
Update Tasklist feedback, ok??

Feel free to change or update it further

### CODE REVIEW

**Comment by beccaelenzil** [comment]
hrmm, this looks good, but it should be a change to `feedback.md` not `readme.md`, right?

File: feedback.md [hunk 1]
```diff
@@ -1,19 +1,64 @@
...
+## Functional Requirements/Manual Testing
+
+| Functional Requirement                                                         | yes/no |
+| ------------------------------------------------------------------------------ | ------ |
+| Successfully handles index & show                                              | ✔️?     |
+| index & show tests pass                                                        | ✔️?     |
+| Successfully handles: New, Create                                              | ✔️?     |
+| New, Create tests pass                                                         | ✔️?     |
+| Successfully handles: Edit, Update                                             | ✔️?     |
+| Edit, Update tests pass with valid & invalid task ids                          | ✔️?     |
```
Comment (jmaddox19) [inline feedback.md]: Can we remove this or move it to a separate "optionals" table?

File: feedback.md [hunk 2]
```diff
@@ -1,19 +1,64 @@
...
+| Successfully handles: Destroy, Task Complete                                   | ✔️?     |
+| Tests for Destroy & Task Complete include tests for valid and invalid task ids | ✔️?     |
```
Comment (jmaddox19) [inline feedback.md]: Can we remove this or move it to a separate "optionals" table?

File: feedback.md (hunk 1 above)
Comment (dHelmgren) [inline feedback.md]:
```suggestion
```

File: feedback.md (hunk 2 above)
Comment (dHelmgren) [inline feedback.md]:
```suggestion
```

File: feedback.md [hunk 3]
```diff
@@ -1,19 +1,61 @@
...
+
+## Overall Feedback
+
+| Overall Feedback                | Criteria                                                                                                                                                                                      | yes/no |
+| ------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------ |
+| Green (Meets/Exceeds Standards) | 5+ in Code Review && 6+ in Functional Requirements                                                                                                                                            |
+| Yellow (Approaches Standards)   | 3+ in Code Review && 5+ in Functional Requirements, or the instructor judges that this project needs special attention                                                                        |
+| Red (Not at Standard)           | 0-2 in Code Review or 0-4 in Functional Reqs, or assignment is breaking/doesn’t run with less than 5 minutes of debugging, or the instructor judges that this project needs special attention |
```
Comment (dHelmgren) [inline feedback.md]:
```suggestion
| Green (Meets/Exceeds Standards) | 5+ in Code Review && 5+ in Functional Requirements                                                                                                                                            |
| Yellow (Approaches Standards)   | 3+ in Code Review && 4+ in Functional Requirements, or the instructor judges that this project needs special attention                                                                        |
| Red (Not at Standard)           | 0-2 in Code Review or 0-3 in Functional Reqs, or assignment is breaking/doesn’t run with less than 5 minutes of debugging, or the instructor judges that this project needs special attention |
```

**Pull request merged**